tools/update-readme.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * @fileoverview Script to update the README with team and sponsors.
+ * Note that this requires eslint/website to be available in the same
+ * directory as the eslint repo.
+ *
+ *   node tools/update-readme.js
+ *
+ * @author Harish Kumar S S
+ */
+
+//-----------------------------------------------------------------------------
+// Requirements
+//-----------------------------------------------------------------------------
+
+import { readFileSync, writeFileSync } from "fs";
```

Comment:
```suggestion
import { readFileSync, writeFileSync } from "node:fs";
```

To fix the lint error.